### DISCUSSION TITLE
Can’t add managed metadata column in details layout of pnp search results webpart

### DISCUSSION THREAD

**Discussion by supreet-bajaj** [microsoft-search/pnp-modern-search#3287] [General]
Hi team,

I am having trouble to add managed metadata property in details layout of pnp search results web part. I am using handlebar expression to populate library column value but it’s showing empty in webpart.

Below is the expression for reference
{{slot item @root.slots.ManagedMetadataName}}.

I have checked in search schema as well and  there is no such managed property available.

Can someone please assist me what’s the process to show additional metadata column in pnp search results web part.

Thanks

**Comment by supreet-bajaj** [comment]
Even I have tried to create new managed property with refinablestring by using column crawled property and waited for some time but still it’s not showing that managed property (refinablestring) in managed columns of pnp search webpart.

attached screenshot is for your reference.

 
![IMG_6227](https://github.com/microsoft-search/pnp-modern-search/assets/25117777/4c212121-aecc-4b06-a63b-9c5c9adfbc7f)

![IMG_6228](https://github.com/microsoft-search/pnp-modern-search/assets/25117777/3d2e73f5-1420-4f1f-87f6-1d723b26c49d)

**Comment by kasperbolarsen** [comment]
Hi , have you tried to use any of the tools in https://microsoft-search.github.io/pnp-modern-search/using-query-tools-to-verify-issues/ ?

Issues with getting the basic mapping to work is not really in scope for this project.
